website/content/docs/ecs/get-started/install.mdx

```diff
@@ -243,7 +243,7 @@ module "my_task" {
 }
 ```
 
-All possible inputs are documented on the [module reference documentation](https://registry.terraform.io/modules/hashicorp/consul/aws-ecs/latest/submodules/mesh-tas?tab=inputs)
+All possible inputs are documented on the [module reference documentation](https://registry.terraform.io/modules/hashicorp/consul-ecs/aws/latest/submodules/mesh-tas?tab=inputs)
```

Comment:
I think there is a 'k' missing for 'mesh-tas' in this link.

I believe it should be 'https://registry.terraform.io/modules/hashicorp/consul-ecs/aws/latest/submodules/mesh-task?tab=inputs'

Reply:
🦅 👀